applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication_mpi.py

```diff
@@ -11,8 +11,8 @@
 except ImportError:
     raise Exception("KratosMPI could not be imported!")
 
-if KratosMPI.mpi.size != 2:
-    raise Exception("The MPI tests currently suport only being run with 2 processors!")
+if KratosMultiphysics.DataCommunicator.GetDefault().Size() != 2:
```

Comment:
Very minor, but you could skip a MPI call with 
```suggestion
if KratosMultiphysics.ParallelEnvironment.GetDefaultSize() != 2:
```